test/unit/DetectChanges/BCBreak/PropertyBased/PropertyDocumentedTypeChangedTest.php

```diff
@@ -113,6 +113,13 @@ class TheClass {
      * @var GenericType<T1, T2>
      */ 
     public $propertyWithComplexDocblockThatCannotBeParsed;
+    
+    /**
+     * @var int
+     */
+    public $propertyWithDocblockTypeHintChangeToNativeTypeHint;
+    
+    public int $propertyWithNativeTypeHintChanged;
```

Comment:
I don't think changes in native types should be handled here: new checker would be better here